lib/mixlib/install/generator/powershell/scripts/helpers.ps1

```diff
@@ -1,5 +1,10 @@
 function Get-PlatformVersion {
-  switch -regex ((get-wmiobject win32_operatingsystem).version) {
+  if(Test-Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server\ServerLevels') {
+    $levels = Get-ItemProperty 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Server\ServerLevels'
+    if($levels.NanoServer -eq 1) { return 'nano' }
```

Comment:
I was thinking about this and planning to change `nano` to `2016nano` based in combo with the actual OS version. The `NanoServer` value above will likely always be `1`. Its just ta flag similar to `ServerCore` but it does appear that releases of nano will be tied to a release of Windows Server. This release is tied to Windows Server 2016.